### ISSUE TITLE
Possible accessibility improvements 

### ISSUE THREAD

**Issue by sonianb** [fac25/unnamed-microblog#31]
I used axe devtools to run accessibility tests and here are some of the issues it flagged:

- The submit button has insufficient colour contrast. The contrast ratio is 3 whereas the expected contrast ratio of 4.5:1.
- The document should have one `main` landmark. [This browser extension](https://matatk.agrip.org.uk/landmarks/) might be helpful to visualise and navigate a webpage via landmarks, even if they're missing. 
- The page must have a level-one heading. 

Hope this helps :sparkles:

**Comment by sonianb** [comment]
By the way, you can use [this contrast checker](https://webaim.org/resources/contrastchecker/) to ensure that the colour contrast ratio for all the elements meets the WCAG standard minimum of 4.5:1.
